src/irepl.cpp

```diff
@@ -96,6 +96,10 @@ main( int argc, char **argv ) {
         exit( 7 );
     }
 
+    if (0 != myRodsArgs.backupMode) {
+        std::printf("Warning: Backup Mode (-B) is deprecated. Its use should be avoided.\n");
+    }
```

Comment:
Do we print out messages about flags being deprecated for other icommands and options?

Reply:
Yes.

https://github.com/irods/irods/issues/4860
https://github.com/irods/irods/issues/7498